docs/index.md

```diff
@@ -19,7 +19,7 @@ S3, or anywhere else you choose.
 ### Great themes available
 
 There's a stack of good looking themes included by default. Choose from
-bootstrap, readthedocs, or any of the 12 bootswatch themes.
+bootstrap, readthedocs, or any of the 12 bootswatch themes. You can also check out a list of 3rd party themes in the [MkDocs wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes), or better yet, add your own.
```

Comment:
Your getting a Markdown-Lint [failure][1] here for line length.

[1]: https://travis-ci.org/mkdocs/mkdocs/jobs/110445743#L196